lib/typo_killer/finder.ex

```diff
@@ -12,7 +12,7 @@ defmodule TypoKiller.Finder do
   def find_typos({words, dictionary, word_map}) do
     candidates =
       words
-      |> Flow.from_enumerable()
+      |> Flow.from_enumerable(max_demand: 100)
```

Comment:
I'd change this function to include the `options \\ []` param, like in the `TypoKiller.FileParser`. This way we can configure it from the CLI:

```elixir
def find_typos({words, dictionary, word_map}, options \\ [])
```

And then get this value with:

```elixir
max_demand = options[:max_demand] || 100
..
```

Then your change would be actually:

```elixir
..
|> Flow.from_enumerable(max_demand: max_demand)
..
```

In `TypoKiller.find_typos/2`, you would change the call to file parser to:

```elixir
..
|> Finder.find_typos(options)
..
```

What to you think?

Reply:
@msramos I like the idea of passing options from the CLI, but I think it could be delivered as a separated issue with another PR

Reply:
@g13ydson but ofc ourse, if you are willing to do it, would be even better 😂

Reply:
@samuelpordeus we don't propagate params from CLI to the pipeline _yet_.

But since we're adding the parameter to the Flow, i think it's pretty easy to just add the params to the function :D